web/models/member.ex

```diff
@@ -21,11 +34,28 @@ defmodule Community.Member do
   def changeset(model, params \\ %{}) do
     model
     |> cast(params, @allowed_fields)
-    |> validate_required(:name)
-    |> validate_required(:email)
-    |> validate_required(:company_name)
+    |> validate_required([:name, :email, :company_name])
+    |> validate_url_format(:website)
+    |> validate_at_least_one_present([:website, :twitter_handle, :dribbble_username], "you must provide at least one social media contact")
+  end
+
+  defp validate_at_least_one_present(changeset, fields, error) do
+    if any_field_present?(changeset, fields) do
+      changeset
+    else
+      add_error(changeset, :social_media, error)
+    end
   end
 
+  defp any_field_present?(changeset, fields) do
+    Enum.any? fields, fn (field) ->
+      changeset |> get_field(field) |> string_length > 1
+    end
+  end
+
+  defp string_length(nil), do: 0
```

Comment:
Maybe this could be `present?` since it also handles `nil` cases and not just strings?